oak_attestation_verification/src/verifier.rs

```diff
@@ -822,6 +840,20 @@ fn extract_root_values(root_layer: &RootLayerEvidence) -> anyhow::Result<RootLay
             })
         }
         TeePlatform::IntelTdx => Err(anyhow::anyhow!("not supported")),
+        TeePlatform::None => {
+            // We use an unsigned, mostly empty AMD SEV-SNP attestation report as a fake when not
```

Comment:
Why AMD SEV-SNP? Should we send something more obviously fake / empty in that case? Just an empty value?

Reply:
This has been the behaviour of Stage 0 for a while. Since Stage 0 currently only supports AMD SEV-SNP this seemed the easiest until we have the need for very different attestation reports such as TDX.

It also keeps the size of the struct the same, which simplifies logic in a few places, and we still populate the "report_data" field so that the DICE chain is rooted in the attestation report even for fake reports.

Created https://github.com/project-oak/oak/issues/4786 to track this.